### PR TITLE
Medium articles bot

### PR DESCRIPTION
**Pull request by Saurav-Shrivastav**
- Modified the Scraper
- Added a new requiremments file for the Medium bot
- Bot is now connected with the Slack RTM Api

### CODE REVIEW

**Comment by Saurav-Shrivastav** [comment]
Please review @shubhank-saxena

File: Medium/scraping.py [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
 
-def search(search, num):
+def search_scrape(search):
     source = requests.get(f"https://www.medium.com/search?q={search}").text
     soup = BeautifulSoup(source, "lxml")
     i = 0
     link = []
     for part in soup.findAll("div", class_="postArticle-readMore"):
-        if i == num:  # Number of articles that need to be fetched
+        if i == 3:  # Number of articles that need to be fetched
```
Comment (shubhank-saxena) [inline Medium/scraping.py:20]: Why 3? I assume we want it to be dynamic in nature? :thinking:

Comment (Saurav-Shrivastav) [reply Medium/scraping.py:20]: That is just a placeholder for now, I have planned a lot more. I have divided the project of the Medium Bot into some more phases, so this issue will be resolved soon!

**Comment by shubhank-saxena** [comment]
Wow! This is looking good to me! :rocket:

**Pull request merged**